src/js/components/AppListComponent.jsx

```diff
@@ -80,8 +76,8 @@ function updateGroupNode(group, app) {
   group.tasksRunning += app.tasksRunning;
   group.totalCpus += app.totalCpus;
   group.totalMem += app.totalMem;
-  group.health = getGroupHealth(group.health, app);
-  group.status = getGroupStatus(group.status, app);
+  group.status = getGroupStatus(group.status, app.status);
```

Comment:
My bad. I only adjusted the order to be consistent with `initGroupNode` property order, and didn't thought about ordering them alphabetically. :innocent: